test/handler-test.js

```diff
@@ -75,26 +75,19 @@ describe('handler', () => {
     });
 
     const result = await handler(event);
-    expect(result).to.match(/inserted 7/i);
-    expect(infos.length).to.equal(5);
+    expect(result).to.match(/inserted 5/i);
```

Comment:
A bunch of the tests changed because the test fixture JSON changed a bit.  Removing any pixel and "preview" records.